upup/pkg/fi/fitasks/keypair.go

```diff
@@ -29,22 +29,28 @@ import (
 )
 
 var wellKnownCertificateTypes = map[string]string{
-	"client": "ExtKeyUsageClientAuth,KeyUsageDigitalSignature",
-	"server": "ExtKeyUsageServerAuth,KeyUsageDigitalSignature,KeyUsageKeyEncipherment",
-	"ca":     "CA,KeyUsageCRLSign,KeyUsageCertSign",
+	"ca":           "CA,KeyUsageCRLSign,KeyUsageCertSign",
+	"client":       "ExtKeyUsageClientAuth,KeyUsageDigitalSignature",
+	"clientServer": "ExtKeyUsageServerAuth,KeyUsageDigitalSignature,ExtKeyUsageClientAuth,KeyUsageKeyEncipherment",
```

Comment:
Sigh - etcd has to use the same cert for both client & server?

Reply:
Indeed ... etcd [tls settings](https://coreos.com/etcd/docs/latest/op-guide/security.html)
```shell
--peer-cert-file=<path>: Certificate used for SSL/TLS connections between peers. 
This will be used both for listening on the peer address as well as sending 
requests to other peers.
```